Stripe/STPAPIClient.m

```diff
@@ -336,8 +334,9 @@ @implementation STPAPIClient (CreditCards)
 
 - (void)createTokenWithCard:(STPCard *)card completion:(STPTokenCompletionBlock)completion {
     NSMutableDictionary *params = [[STPFormEncoder dictionaryForObject:card] mutableCopy];
-    params[@"muid"] = [STPAnalyticsClient muid];
+    [[STPTelemetryClient sharedInstance] addTelemetryFieldsToParams:params];
     [self createTokenWithParameters:params completion:completion];
```

Comment:
Is there a reason we are not doing this inside createTokenWithParameters? Do we not want telemetry on non-card tokenizations?